packages/desktop/components/popups/send/SendTokenForm.svelte

```diff
@@ -0,0 +1,168 @@
+
+            return Promise.resolve()
+        } catch (err) {
+            handleError(err)
+            return Promise.reject()
+        } finally {
+            isPreparingOutput = false
+        }
+    }
+
+    async function onContinueClick(): Promise<void> {
+        const valid = await validateInputs(
+            [(): Promise<void> => assetAmountInput?.validate(false), recipientInput?.validate, networkInput?.validate],
+            [
+                {
+                    validate: metadataInput?.validate,
+                    value: metadata,
+                    optionalInputType: OptionalInputType.Metadata,
+                    byteLimit: MAX_METADATA_BYTES,
+                    errorMessage: localize('error.send.metadataTooLong'),
+                },
+                {
+                    validate: tagInput?.validate,
+                    value: tag,
+                    optionalInputType: OptionalInputType.Tag,
+                    byteLimit: MAX_TAG_BYTES,
+                    errorMessage: localize('error.send.tagTooLong'),
+                },
+            ]
+        )
+
+        if (valid) {
+            await buildPreparedOutput()
+
+            openPopup({
+                id: PopupId.SendConfirmation,
+                overflow: true,
+                props: {
+                    preparedOutput,
+                },
+            })
+        }
+    }
+
+    function onCancelClick(): void {
+        closePopup()
+    }
+</script>
+
+<send-token-form class="flex flex-col space-y-4">
+    <send-form-inputs class="flex flex-col space-y-4">
+        <AssetAmountInput bind:this={assetAmountInput} bind:asset bind:rawAmount bind:unit {disableAssetSelection} />
+        <NetworkInput bind:this={networkInput} bind:iscpChainAddress {showLayer2} />
+        <RecipientInput bind:this={recipientInput} bind:recipient isLayer2={isLayer2Transfer} />
+        <optional-inputs class="flex flex-row flex-wrap gap-4">
+            <OptionalInput
+                bind:this={tagInput}
+                bind:value={tag}
+                label={localize('general.tag')}
+                description={localize('tooltips.optionalInput')}
+            />
+            {#if !isLayer2Transfer}
+                <OptionalInput
+                    bind:this={metadataInput}
+                    bind:value={metadata}
+                    label={localize('general.metadata')}
+                    description={localize('tooltips.optionalInput')}
+                />
+            {/if}
+        </optional-inputs>
+    </send-form-inputs>
+    {#if isTransferring}
+        <Error error={localize('notifications.transferring')} />
+    {/if}
+    <popup-buttons class="flex flex-row flex-nowrap w-full space-x-4">
+        <Button classes="w-full" outline onClick={onCancelClick}>
+            {localize('actions.cancel')}
+        </Button>
+        <Button
+            classes="w-full"
+            onClick={onContinueClick}
```

Comment:
No validation errors are shown to user when clicking onContinue when amount is left as 0. Error is logged in the console but not shown to the user
![image](https://github.com/iotaledger/firefly/assets/52735957/010d5dfe-1114-4305-999f-b1d7ffe40280)

Reply:
As written in the PR description, we should fix it in a separate PR. The AssetAmountInput validation doesn't work properly like other inputs.